test/test_call_lua.lua

```diff
@@ -29,10 +30,39 @@ end
 function TestCallLua:printTable(sometable, somestring, table2)
   for k, v in pairs(sometable) do
     print('TestCallLua:printTable ', k, v)
+    if k == 'color' then
+        assert(v == 'red')
+    elseif k == 'weather' then
+        assert(v == 'sunny')
+    elseif k == 'anumber' then
+        assert(v == 10)
+    elseif k == 'afloat' then
+        assert((v - 1.234) < 1e-7)
+    end
   end
+  assert(somestring == 'mistletoe')
   print('somestring', somestring)
   for k, v in pairs(table2) do
     print('TestCallLua table2 ', k, v)
+    if k == 'row1' then
+        assert(v == 'col1')
+    elseif k == 'meta' then
+        assert(v == 'data')
+    end
   end
   return {bear='happy', result=12.345, foo='bar'}
 end
+
+function TestCallLua:getList(somelist)
+  assert(type(somelist) == 'table')
+  print('list in lua:')
+  for i = 1, #somelist do
+    print(i, somelist[i])
+  end
+  assert((somelist[1] - 3.1415) < 1e-7)
+  assert(somelist[2] == '~Python\\omega')
+  assert(somelist[3] == 42)
+
+  table.insert(somelist, 'Lorem Ipsum')
```

Comment:
I'd prefer to allocate a brandnew list on the whole.  The way the python/lua boundary works involves registering objects as they move in each direction, and the registration process seems to work on the whole, but I'd prefer to doublecheck on the whole for lists.  I guess it should work since it's re-using how dictionaries work?  But would be nice to doublecheck :-)